com.endlessm.Clippy.Extension.json.in

```diff
@@ -16,9 +16,22 @@
       ],
       "sources": [
         {
-          "type": "git",
-          "path": ".",
-          "branch": "@GIT_CLONE_BRANCH@"
+          "type": "dir",
+          "path": "."
+        }
+      ]
+    },
+    {
+      "name": "appdata",
```

Comment:
Instead of here, this should be installed from the meson build process.

Reply:
That would be the case in a normal app. But I checked how the Intel drivers extension is doing it and adapted it here.

Reply:
I am not sure what the Intel drivers extension has to do with this? Is there any reason why you would not want to change it to be installed by the meson build process?

Reply:
Full story: I did try to install it through the build system. I saw it was still using the `/app` prefix and would hit "readonly" when trying to copy it there, so I went to look what the "right" way or path was by checking a couple of extensions and found out they were doing it directly in the manifest and went to that since I wanted to move on.
This is worth another attempt if we're not in a rush anymore, so I am fine trying again tomorrow.

Reply:
Oh, I see why it's done like that now. I think it's fine then!